quiztest/src/main/java/quiztest/main/inventory/InventoryService.java

```diff
@@ -0,0 +1,220 @@
+        Timestamp curr = new Timestamp(System.currentTimeMillis());
+        logger.info("Scheduled Task. Purchasing random vehicle {} at {}", vehicle.toString(), curr);
+        inventoryRepositoryJDBC.purchaseRandomVehicle(vehicle.getVIN(), curr);
+    }
+
+    public Vehicle findByVIN(String VIN) {
+        logger.info("Searching for vehicles by VIN. VIN = {} ", VIN);
+        return inventoryRepositoryJDBC.findByVIN(VIN);
+    }
+
+    public Vehicle createRandomVehicle() {
+        logger.info("Creating a random vehicle.");
+        Random random = new Random();
+        Vehicle vehicle = new Vehicle();
+
+        Pair<String, List<String>> beep = makeModelList.get(random.nextInt(makeModelList.size()));
+
+        vehicle.setMake(beep.getLeft());
+        vehicle.setModel(beep.getRight().get(random.nextInt(beep.getRight().size())));
+        vehicle.setAvailable(true);
+        vehicle.setColor(colorList.get(random.nextInt(colorList.size())));
+        vehicle.setVIN(RandomStringUtils.randomAlphanumeric(10).toUpperCase());
+        vehicle.setLocation(Integer.parseInt(RandomStringUtils.randomNumeric(1)) + 1);
+        return vehicle;
+    }
+
+    public Vehicle createRandomSoldVehicle() {
+        logger.info("Creating a random sold vehicle.");
+        Random random = new Random();
+        Vehicle vehicle = new Vehicle();
+
+        long offset = Timestamp.valueOf("2021-01-01 00:00:00").getTime();
+        long end = Timestamp.valueOf("2021-05-01 00:00:00").getTime();
+        long diff = end - offset + 1;
+        Timestamp ts = new Timestamp(offset + (long) (Math.random() * diff));
+
+        Pair<String, List<String>> beep = makeModelList.get(random.nextInt(makeModelList.size()));
+
+        vehicle.setMake(beep.getLeft());
+        vehicle.setModel(beep.getRight().get(random.nextInt(beep.getRight().size())));
+        vehicle.setAvailable(false);
+        vehicle.setColor(colorList.get(random.nextInt(colorList.size())));
+        vehicle.setVIN(RandomStringUtils.randomAlphanumeric(10).toUpperCase());
+        vehicle.setLocation(Integer.parseInt(RandomStringUtils.randomNumeric(1)) + 1);
+        vehicle.setDateOfSale(ts);
+        return vehicle;
+    }
+
+    public boolean exists(Vehicle vehicle) {
+        if (findByVIN(vehicle.getVIN()) != null) {
+            logger.info("{} exists.", vehicle.getVIN());
+            return true;
+        }
+        logger.info("Proposed Vehicle does not exist yet.");
+        return false;
+
+    }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+
+        logger.info("Making model list");
+
+        makeModelList.add(Pair.of("Ford", Arrays.asList("Mustang", "Taurus")));
+
+        makeModelList.add(Pair.of("Tesla", Arrays.asList("Model 3", "Model S")));
+
+        makeModelList.add(Pair.of("Honda", Arrays.asList("Accord", "Fit")));
+
+        makeModelList.add(Pair.of("Nissan", Arrays.asList("350z", "Sentra")));
+
+    }
+
+    @Scheduled(cron = "0 0 * * * *")
```

Comment:
A bit of a nitpick, but you should move this method **above** the afterPropertiesSet() method so afterPropertiesSet is the last method in this class.